k8s/demo/common/ccd/latest-ccd-chart.yaml

```diff
@@ -38,6 +38,7 @@ spec:
             secretRef: ccd-api-gateway-oauth2-client-secret
             key: key
```

Comment:
Hi @hemantt, it would be better if we deployed this on a separate CCD full stack deployment dedicated to Pay team. Otherwise we are stuck with this version of management web and we might need to change back to master. 
@akdasari is working on a feature that allows you to deploy multiple full stack isolated instances of CCD on AKS Demo. Each with it's own name. It's going to be ready by tomorrow I think. Can you wait tomorrow?

Reply:
This is a temporary request by Pay team to verify a beta change on AKS. Will be rolled back to master next week, until a final release on to Production